words/disasm.asm

```diff
@@ -40,151 +40,109 @@ z_disasm:       rts
 
 
 disassembler:
-                jsr w_cr       ; ( addr u )
+                jsr w_cr            ; ( addr u )
 _byte_loop:
                 ; Print address at start of the line. Note we use whatever
                 ; number base the user has
-                jsr w_over     ; ( addr u addr )
-                jsr w_u_dot    ; ( addr u )
+                jsr w_over          ; ( addr u addr )
+                jsr w_u_dot         ; ( addr u )
                 jsr w_space
 
-                ; We use the opcode value as the offset in the oc_index_table.
-                ; We have 256 entries, each two bytes long, so we can't just
-                ; use an index with Y. We use tmp2 for this.
-                lda #<oc_index_table
+                lda 2,x
                 sta tmp2
-                lda #>oc_index_table
+                lda 3,x
                 sta tmp2+1
 
-                lda (2,x)       ; get opcode that addr points to
-                sta scratch     ; Save opcode
+                lda (tmp2)          ; get opcode that addr points to
+                sta scratch         ; save a copy
+                jsr op_find_nt      ; get NT or 0 in tmp1
```

Comment:
This is simpler to read now, although a little slower since it searches for each opcode in the assembler word list rather than using a direct lookup table.